.github/workflows/cicd.yaml

```diff
@@ -127,13 +98,14 @@ jobs:
           name: coverage-data
           path: .tox
 
-      - name: list coverage files
-        run: |
-          find .tox -name ".coverage.*"
+      # - name: list coverage files
```

Comment:
Any chance we can uncomment this? I think it was added for information purposes and also in preparation for adding the coverage badge 🤔 so we know what files can be accessed.

Reply:
Recovered!